examples/VLCB_4in4out/VLCB_4in4out.ino

```diff
@@ -46,21 +46,18 @@
 
 // 3rd party libraries
 #include <Streaming.h>
-#include <Bounce2.h>
 
 // VLCB library header files
 #include <VLCB.h>
 #include <CAN2515.h>               // Chosen CAN controller
 
-#include "LEDControl.h"
-
 // forward function declarations
 void eventhandler(byte, const VLCB::VlcbMessage *);
 void printConfig();
 void processSwitches();
 
 // constants
-const byte VER_MAJ = 1;             // code major version
+const byte VER_MAJ = 2;             // code major version
```

Comment:
Not a functional change. I'd say you only need to update the minor version letter.

Reply:
OK.  I wondered but updated because the support classes and library had changed.  I'll change to 1b